### PR TITLE
[META] More fixes

### PR DESCRIPTION
**Pull request by ejgallego**
- install Coq's VM libraries in the same location `kernel/byterun` for both `-local` and non-local builds. This allows to have a single META file for both cases in native linking.
- adapt to Ltac's pluginification
- change version number.

### CODE REVIEW

**Comment by gares** [comment]
I think we should have a META.in (as many OCaml projects do) and substitute a few variables in it..

**Comment by ejgallego** [comment]
> I think we should have a META.in (as many OCaml projects do) and substitute a few variables in it..

Maybe, but note that the current solution works fine. What variables would you need to substitute?

**Comment by gares** [comment]
The commit replaces many instances of 8.6 by 8.7 ...

**Comment by ejgallego** [comment]
> The commit replaces many instances of 8.6 by 8.7 ...

Oh good point. Added to the TODO list.

Any particular suggestion of tool for the `META.in -> META` transform? Do we have a recommended method?

**Comment by gares** [comment]
Dunno what configure.ml uses to generate its files.
I also have some code in coq_makefile2 for that based on the `Str` module, I would just copy that one (look for `@` in CoqMakefile.in)

**Comment by ejgallego** [comment]
@gares indeed I was a bit concerned about rolling out yet another macro replacement system, and my usual choice may be problemactic for windows. Most OCaml projects use autotools so that is easy for them. I'll have a look to what coq_makefile2 does.

**Comment by ejgallego** [comment]
Indeed, it would make sense to reuse the same coq_makefile2 mechanism, not only for the pattern substitution, but also it should integrate sourcing of the variables value.

I've update META to add a comment reflecting this TODO, meanwhile, this is IMHO ready to go.

**Comment by ejgallego** [comment]
In fact it was already noted "this file should be generated a configure time". However it works now for all the cases except a local build + bytecode.

File: configure.ml [hunk 1]
```diff
@@ -926,7 +926,7 @@ let config_runtime () =
   | _ ->
     let ld="CAML_LD_LIBRARY_PATH" in
     build_loadpath := sprintf "export %s:='%s/kernel/byterun':$(%s)" ld coqtop ld;
-    ["-dllib";"-lcoqrun";"-dllpath";libdir]
+    ["-dllib";"-lcoqrun";"-dllpath";libdir/"kernel/byterun"]
```
Comment (maximedenes) [inline configure.ml]: Can you say more about this change?

Comment (ejgallego) [inline configure.ml]: This is used to build the bytecode executable, and as you can see it was different before in the local case. Now the vm is always installed in `coqlib/kernel/byterun`.

**Pull request merged**